target_postgres/db_sync.py

```diff
@@ -29,7 +29,8 @@ def column_type(schema_property):
         return 'character varying'
 
 
-def inflect_column_name(name):
+def inflect_name(name):
+    name = name.replace(' ', '_')
```

Comment:
Column and sheet names from the google sheets tap are user-defined, right? If so, that opens up a whole can of worms here.

Up until now, the set of column and table names produced by all taps is pretty small. With arbitrary user input, we are going to need to be much more defensive in this function:

- Is it possible for there to be duplicate column or table names sent from the tap?
  - If so, it should probably trigger an error, but errors from tap land don't make it to the user, so that's a bad experience. I'm not really sure if there's a good solution to this.
- Could column or table names contain any arbitrary utf8 string?
  - If so, there a lot of other characters we need to look at replacing.
  - By modifying the name, we introduce the possibility of having duplicate column or table names. This will be much more likely to happen with arbitrary user input (the variety of column and table names will go way up).
- Could a column name or table name be empty?